ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/repo/migrator.ex

```diff
@@ -0,0 +1,45 @@
+defmodule ExCubicOdsIngestion.Repo.Migrator do
+  @moduledoc """
+  GenServer which runs on startup to run Ecto migrations, then terminates.
+  """
+  require Logger
+
+  @opts [module: Ecto.Migrator, run_migrations_at_startup?: true]
+
+  @spec start_link(Keyword.t()) :: :ignore
+  def start_link(opts) do
+    opts = Keyword.merge(@opts, opts)
+
+    if Keyword.get(opts, :run_migrations_at_startup?) do
+      _ignored = Logger.info("Migrating synchronously")
```

Comment:
suggestion: I don't think you need `_ignored = ` anymore, as recent versions of Elixir have `Logger.info` always returning `:ok`.
```suggestion
      Logger.info("Migrating synchronously")
```